content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -192,51 +235,75 @@ If you are on macOS and using [Macports](https://macports.org/) package manager,
 
 ### Install kubectl binary with curl on Windows
 
-1. Download the latest release {{< param "fullversion" >}} from [this link](https://storage.googleapis.com/kubernetes-release/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe).
+1. Download the latest release, {{< param "fullversion" >}}, from [this link](https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe).
```

Comment:
nit: Do you need to add the commas?
nit: Could you add text to identify the link instead of "this link"?

Reply:
> nit: Do you need to add the commas?

I think I do--I suppose I'm using the release version as an interrupter. I could use parentheses, or restructure the sentence. I'm not actually sure we need the version identified there now that I'm looking at it again. I'm happy to take direction on this one though.

> nit: Could you add text to identify the link instead of "this link"?

Yes, and I think I can solve the preceding nit in the same edit.

```suggestion
1. Download the [latest release {{< param "fullversion" >}}](https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe).
```

Reply:
https://deploy-preview-25259--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/install-kubectl/#install-kubectl-binary-with-curl-on-windows